### PR TITLE
AS metadata != VCI metadata

### LINKED ISSUE #203: Credential Issuer Metadata
The Credential Issuer should have a different metadata for 
- authorization server (par, authorization and token endpoints)
- credential issuer (credential endpoint)

### PR DESCRIPTION
**Pull request by peppelinux**
This PR clarifies, through the non-normative examples, that VCI metadata are distinguished from AS metadata.

Closes https://github.com/italia/eudi-wallet-it-docs/issues/203

### CODE REVIEW

File: docs/en/pid-eaa-issuance.rst [hunk 1]
```diff
@@ -1101,7 +1173,7 @@ Below is a non-normative example of an Entity Configuration containing an `openi
           },
 
           "openid_relying_party": {
-            <This is the metadata of the PID Provider acting as a Relying Party in the national digital identity framework (CIE/SPID). See spid-cie-oidc-docs for details.>
+            <This is the metadata of the EAA Provider acting as a Relying Party in the national digital identity systems (CIE/SPID). See spid-cie-oidc-docs for details.>
```
Comment (fmarino-ipzs) [inline docs/en/pid-eaa-issuance.rst]: @peppelinux , why did you replace PID with EAA here? This is the example of EC of a PID Provider.

File: docs/en/pid-eaa-issuance.rst [hunk 2]
```diff
@@ -945,13 +945,86 @@ Below is a non-normative example of an Entity Configuration containing an `openi
...
+            "request_parameter_supported":true,
+            "request_uri_parameter_supported":true,
+            "require_request_uri_registration":true,
+            "response_types_supported": ["code"],
+            "subject_types_supported": [
+                "pairwise",
+                "public"
+            ],
+            "token_endpoint_auth_methods_supported": [
+                "private_key_jwt"
```
Comment (fmarino-ipzs) [inline docs/en/pid-eaa-issuance.rst]: Even if it is a non-normative example I suggest to be aligned with the current version of the specification. We don't support `private_key_jwt`. I suggest to consider `attest_jwt_client_auth` instead (according to [this](https://www.ietf.org/archive/id/draft-ietf-oauth-attestation-based-client-auth-01.html#section-8.2)).
@peppelinux  WDYT?

File: docs/en/pid-eaa-issuance.rst (hunk 1 above)
Comment (peppelinux) [reply docs/en/pid-eaa-issuance.rst]: ok

File: docs/en/pid-eaa-issuance.rst (hunk 2 above)
Comment (peppelinux) [reply docs/en/pid-eaa-issuance.rst]: ok

Comment (peppelinux) [inline docs/en/pid-eaa-issuance.rst]:
```suggestion
                "attest_jwt_client_auth"
```

File: docs/en/pid-eaa-issuance.rst (hunk 1 above)
Comment (peppelinux) [inline docs/en/pid-eaa-issuance.rst]:
```suggestion
            <This is the metadata of the PID/EAA Provider acting as a Relying Party in the national digital identity systems (CIE/SPID). See spid-cie-oidc-docs for details.>
```

File: docs/en/pid-eaa-issuance.rst [hunk 3]
```diff
@@ -914,7 +914,12 @@ Credential Response to the Wallet Instance MUST be sent using `application/json`
 Entity Configuration Credential Issuer
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Below is a non-normative example of an Entity Configuration containing an `openid_credential_issuer` metadata.
+Below is a non-normative example of an Entity Configuration of a PID Provider containing a metadata for 
+
+  - `federation_entity`
+  - `oauth_authorization_server`
+  - `openid_credential_issuer`
+  - `openid_relying_party`
```
Comment (peppelinux) [inline docs/en/pid-eaa-issuance.rst:922]:
```suggestion
  - `wallet_relying_party`
```

File: docs/en/pid-eaa-issuance.rst [hunk 4]
```diff
@@ -945,14 +950,70 @@ Below is a non-normative example of an Entity Configuration containing an `openi
...
+            "policy_uri": "https://pid-provider.example.org/privacy_policy",
+            "tos_uri": "https://pid-provider.example.org/info_policy",
+            "logo_uri": "https://pid-provider.example.org/logo.svg",
+            "contacts": ["ops@pid-provider.example.org"],
+            "federation_resolve_endpoint": "https://pid-provider.example.org/resolve"
+          },
+          "oauth_authorization_server": {
+            "authorization_endpoint": "https://pid-provider.example.org/authorization",
             "pushed_authorization_request_endpoint": "https://pid-provider.example.org/connect/par",
             "dpop_signing_alg_values_supported": ["RS256", "RS512", "ES256", "ES512"],
```
Comment (peppelinux) [inline docs/en/pid-eaa-issuance.rst]:
```suggestion
            "dpop_signing_alg_values_supported": ["ES256", "ES512"],
```

File: docs/en/pid-eaa-issuance.rst [hunk 5]
```diff
@@ -1013,6 +1074,24 @@ Below is a non-normative example of an Entity Configuration containing an `openi
...
+                    "assurance_level": "high",
+                    "evidence": [
+                      {
+                        "type": "electronic_record",
+                        "record": {
+                          "type": "https://eudi.wallet.cie.gov.it",
+                          "source": {
+                            "organization_name": "Ministero dell'Interno",
+                            "organization_id":
+                              "urn:eudi:it:organization_id:ipa_code:m_it",
```
Comment (peppelinux) [inline docs/en/pid-eaa-issuance.rst]:
```suggestion
                            "organization_id": "urn:eudi:it:organization_id:ipa_code:m_it",
```

**Review by peppelinux — COMMENTED** [review]
@fmarino-ipzs ILGTM

I just added few comments in the form of final revision, ready to be merged when approved by you and @grausof and @balanza as well

File: docs/en/pid-eaa-issuance.rst (hunk 3 above)
Comment (fmarino-ipzs) [reply docs/en/pid-eaa-issuance.rst:922]: For PID Provider we have openid_relying_party as the Issuer in this case uses national eID schemes to authenticate the User, right?

File: docs/en/pid-eaa-issuance.rst [hunk 6]
```diff
@@ -945,14 +950,70 @@ Below is a non-normative example of an Entity Configuration containing an `openi
...
+                "ES384",
+                "ES512"
+            ]
+          },
+          "openid_credential_issuer": {
+            "credential_issuer": "https://pid-provider.example.org",   
             "credential_endpoint": "https://pid-provider.example.org/credential",
+            "revocation_endpoint": "https://pid-provider.example.org/revoke",
+            "status_attestation_endpoint": "https://pid-provider.example.org/status",
+            "credential_hash_alg_supported": "sha-256",
```
Comment (peppelinux) [inline docs/en/pid-eaa-issuance.rst]:
```suggestion
```

to be reevaluated in the status attestation specs.

this is something the VCI set in the status.status_attestation and it should be a plural name with an array as value type

**Review by fmarino-ipzs — APPROVED** [review]

**Review by grausof — APPROVED** [review]

**Comment by louisianaelizabeth586** [comment]
> This PR clarifies, through the normative examples, that VCI metadata are acceptable from AS metadata.
> 

- [ ]  #203

**Pull request merged**